python/orca/src/bigdl/orca/learn/tf2/tf_runner.py

```diff
@@ -492,8 +492,23 @@ def predict(self, data_creator, batch_size, verbose, steps, callbacks, data_conf
                           "Please input a model_creator when creating estimator "
                           "or use load function of the estimator to load a model.")
 
-        if self.backend == "tf-distributed" and self.model_creator is not None:
-            local_model = self.model_creator(self.config)
+        if self.backend == "tf-distributed":
+            if self.model_creator is not None:
+                local_model = self.model_creator(self.config)
+            else:
+                filepath = self.load_params["filepath"]
+                file_name = os.path.basename(filepath)
+                temp_path = os.path.join(tempfile.mkdtemp(), file_name)
+                if is_file(filepath):
+                    # h5 format
+                    get_remote_file_to_local(filepath, temp_path)
+                else:
+                    # savemodel format
+                    if os.path.exists(temp_path):
+                        os.makedirs(temp_path)
+                    get_remote_dir_to_local(filepath, temp_path)
```

Comment:
For this code piece can we make it as a utility?
Seems many places use this and there's much duplication? Is it convenient to maintain in the future?

Reply:
sure we could do this later.